spec/duration.html

```diff
@@ -1455,7 +1474,10 @@ <h1>
             1. Set _oneMonthDays_ to _moveResult_.[[Days]].
         1. Else,
           1. Assert: _largestUnit_ is *"week"*.
-          1. Let _dateAdd_ be ? GetMethod(_calendar_, *"dateAdd"*).
+          1. If _calendar_ is an Object, then
+            1. Let _dateAdd_ be ? GetMethod(_calendar_, *"dateAdd"*).
+          1. Else,
+            1. Let _dateAdd_ be ~unused~.
```

Comment:
This will be completely different in #2519 anyway, and I already have a note to myself on that topic: https://github.com/tc39/proposal-temporal/pull/2519#discussion_r1166146921
I'd prefer not to introduce an operation here, but instead focus on #2519.